webapp/graphql/User.js

```diff
@@ -283,3 +283,12 @@ export const currentUserQuery = gql`
     }
   }
 `
+
+export const currentUserCountQuery = () => gql`
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-27T11:06:01Z" title="Wednesday, May 27th 2020, 1:06:01 pm +02:00">May 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -283,3 +285,12 @@ export const currentUserQuery = gql`
     }
   }
 `
+
+export const currentUserCountQuery = () => gql`
```


Great you did a separate query !!! 😍
The performance is why you did it separate I guess. TOP!

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-07-30T11:41:05Z" title="Thursday, July 30th 2020, 1:41:05 pm +02:00">Jul 30, 2020</time>_

---


ok